submissions/ArthurGorbenko/Friend's app/js/script.js

```diff
@@ -0,0 +1,282 @@
+const CONSTS = {
+  LIST_OF_CARDS: document.querySelector(".list"),
+  DIRECT_MESSAGE_FORM: document.querySelector(".direct-message"),
+  FORM: document.querySelector(".form"),
+  SEND_BUTTON: document.querySelector(".send-on-click-button"),
+  SEARCH_FIELD: document.querySelector(".search-field"),
+  RADIO_AGE_FORM: document.querySelector(".wrapper-radio-age"),
+  RADIO_GENDER_FORM: document.querySelector(".wrapper-radio-gender"),
+  EMAIL_FIELD: document.querySelector(".email-field"),
+  TEXT_AREA: document.querySelector(".textarea"),
+  ALERT_MESSAGE: document.querySelector(".alert-message"),
+  RESET_BUTTON: document.querySelector(".reset"),
+  HAMBURGER: document.querySelector(".hamburger.hamburger--slider"),
+  THEME_WRAPPER: document.querySelector(".wrapper-theme-change"),
+  BODY: document.querySelector(".default"),
+  MENU: document.querySelector(".menu"),
+  WRAPPER_MENU: document.querySelector(".wrapper-menu")
+};
+
+const MOBILE_RESOLUTION = "(max-width: 599px)";
+const SORT_BY_ASCENDING = "ascending";
+const CARD_CLASS = "card";
+const BUTTON_CLASS = "button";
+const DARK_INPUT_ID = "dark";
+
+let filteredByGender = false;
+let radioButtons = document.querySelectorAll(".visually-hidden");
+let usersConst;
+let users;
+
+const letterToUpperCase = word => {
+  return word.charAt(0).toUpperCase() + word.slice(1);
+};
+
+const refreshDMWindow = () => {
+  CONSTS.FORM.classList.remove("visually-hidden");
+  CONSTS.ALERT_MESSAGE.classList.add("visually-hidden");
+  CONSTS.TEXT_AREA.value = "";
+};
+
+let menuToggled = false;
+
+const moveMenu = () => {
+  let position = 0;
+  let middleScreen = Math.floor(
+    (window.innerWidth - CONSTS.WRAPPER_MENU.clientWidth) / 2
+  );
+  let id = setInterval(relocate, 1);
+  function relocate() {
+    if (position === middleScreen) {
+      clearInterval(id);
+    } else {
+      position++;
+      CONSTS.WRAPPER_MENU.style.left = position + "px";
+    }
+  }
+  menuToggled = true;
+};
+
+const moveContent = () => {
```

Comment:
Also, try do this with CSS. 
One more things - try to not change styles where it can be changed by CSS. 
Adding classes the best option.

Reply:
Done.